parts/k8s/artifacts/kubernetesazurekms.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=azurekms
+Requires=docker.service
+After=network-online.target
+
+[Service]
+Type=simple
+Restart=always
+TimeoutStartSec=0
+ExecStart=/usr/bin/docker run \
+  --net=host \
+  --volume=/opt:/opt \
+  --volume=/etc/kubernetes:/etc/kubernetes \
+  --volume=/etc/ssl/certs/ca-certificates.crt:/etc/ssl/certs/ca-certificates.crt \
+  --volume=/var/lib/waagent:/var/lib/waagent \
+  ritazh/k8s-azure-kms:latest
```

Comment:
@khenidak what do we need to do to (1) publish this to the `Azure` org, and (2) ensure that we maintain that image over time?

Reply:
@ritazh ☝️ fyi